.github/workflows/maven.yml

```
@@ -41,9 +41,15 @@
       with:
         java-version: ${{ matrix.java-version }}
     - name: Build with Maven
-      run: ./mvnw clean install -Dmaven.test.skip=true -B -U -e && bash ./.middleware-common/check_format.sh
+      run:   if [[ ${{ matrix.java-version }} -gt 17 ]]; then
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1046:error:1:1: Couldn't find 'fi' for this 'if'

(shellcheck)